### ISSUE TITLE
Make it so that "Temporarily Reveal" reveals the CP in the forecast maker as well as the graph

### ISSUE THREAD

**Issue by hickman-santini** [Metaculus/metaculus#602]
**TL;DR:** @lsabor implemented a fix for https://github.com/Metaculus/metaculus/issues/426 that means people with "Show Community Prediction by default on open questions" toggled "off" in their settings can't see the full distribution on numeric questions even when they click _Temporarily Reveal_...

More details from the original issue:

> the method to reveal the CP is embedded in the graph component which would then have to talk with the forecast input component, which is not trivial and would require some refactoring.
I was expecting that when the CP is hidden we don't actually return populated CP values and we could just retrieve the values at will if we want to temporarily reveal it, but instead it looks like we have the data and hide it and so it will require some more complicated state management.

> A future refactor will be required, but this does allow users with the setting on to successfully hide it from themselves for now. Moving to medium priority for addressing the straggling issue as it's no longer blocking.
I'd still like to just not return the CP details at all with the setting on and never make any checks about displaying - there are too many locations to add the conditions.

**Comment by nangchrys** [comment]
I think this has been fixed.

**Comment by lsabor** [comment]
closing as complete
